nifi-api/src/main/java/org/apache/nifi/reporting/EventAccess.java

```diff
@@ -63,4 +63,32 @@
      */
     List<Action> getFlowChanges(int firstActionId, final int maxActions);
 
+    /**
```

Comment:
@mattyb149 wondering if this interface can be simplified? In the implementation class (RingBufferEventRepository) the aggregatedValues object is used to produce these values by first converting that object to a flowFileEvent.  Can we  instead do a method like "reportAggregatedEvents" which returns a FlowFileEvent?  Then callers could just use that single FlowFileEvent to do getBytesRead/getBytesWritten etc.

Reply:
Makes sense, but FlowFileEvent is in nifi-framework-core-api, so we'd likely have to move that into nifi-api so everyone can use it. I'll look into the best way to do that.

Reply:
@mattyb149 my apologies I meant to mark my comment on the FlowFileEventRepository, not EventAccess.